### PR TITLE
Pract.04: Add 01-22; Pract.05: Add 01-12   | Alex Hristov

### PR DESCRIPTION
**Pull request by alexhr05**

### CODE REVIEW

**Comment by alexhr05** [comment]
Alex Hristov
3 6MI0600471

File: Sem.04/Pract.04/Alex Hristov/Zad_17.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        i++;
+    }while(pow(2,i)<num);
+    if(abs(pow(2,i)-num) > abs(pow(2,i+1)-num)){
+        return pow(2,i+1);
+    }else{
+        return pow(2,i);
+    }
+    
+}
+int main()
```
Comment (SirLex) [inline Sem.04/Pract.04/Alex Hristov/Zad_17.cpp:21]: Please leave some spaces and new lines after logical stops,  end of functions and so on

File: Sem.04/Pract.04/Alex Hristov/Zad_2.cpp [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+using namespace std;
+
+bool isEven(int a){
+    if(a%2==0){
```
Comment (SirLex) [inline Sem.04/Pract.04/Alex Hristov/Zad_2.cpp:5]: a%2==0 is a boolean expression 

Cant we just return it

File: Sem.04/Pract.04/Alex Hristov/Zad_3_secondPart.cpp [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+using namespace std;
+
+bool isNegative(int a){
+    if(a<0){
+        return true;
```
Comment (SirLex) [inline Sem.04/Pract.04/Alex Hristov/Zad_3_secondPart.cpp:6]: Cant we just return a<o

File: Sem.04/Pract.04/Alex Hristov/Zad_6.cpp [hunk 4]
```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+using namespace std;
+
+bool isCapitalLetter(char S){
+    if(S>= 'A' && S<='Z'){
```
Comment (SirLex) [inline Sem.04/Pract.04/Alex Hristov/Zad_6.cpp:5]: Returning boolean expresion

File: Sem.05/Pract.05/Alex Hristov/Zad_8.cpp [hunk 5]
```diff
@@ -0,0 +1,49 @@
+#include <iostream>
+using namespace std;
+
+const int n = 1000;
+
```
Comment (SirLex) [inline Sem.05/Pract.05/Alex Hristov/Zad_8.cpp:5]: Very bad formatting of code. My eyes hurt

**Review by SirLex — COMMENTED** [review]
Overall great logic wise.
I have spoted a lot of places where you check boolean expresion in if instead of just returning it.

Also you should format more of your code, new lines, correct identation and so on

**Pull request merged**